use_case_examples/lora_finetuning/README.md

```diff
@@ -47,14 +67,18 @@ In a deployment or production scenario, the model can be fine-tuned as follows:
 
 ## Results
 
-The fine-tuned model can generate specialized text based on the provided data-set while ensuring data privacy through FHE.
 
+### GPT-2 Results
 After fine-tuning, the model's weights are distributed between the client and server as follows:
 
 - Total weights removed from the server: 68.24%
 - LoRA weights kept on the client: 147,456 (approximately 0.12% of the original model's weights)
 
-Note that the embedding are not considered for now but contain a significant amount of weights (around 30%) for GPT2. They will be considered in a future version of Concrete ML.
+Note that the embeddings are not considered for now but contain a significant amount of weights (around 30%) for GPT2. They will be considered in a future version of Concrete ML.
+
+### LLaMA Results
+
+TBD
```

Comment:
@jfrery, i think you forgot that part

Reply:
No we don't have the result yet. it's a WIP